test/boost/user_types_test.cc

```diff
@@ -580,7 +580,7 @@ SEASTAR_TEST_CASE(test_nonfrozen_user_types_prepared) {
         BOOST_REQUIRE_EXCEPTION(
             execute_prepared("insert into cf (a, b) values (?, ?)", {mk_int(4), mk_longer_tuple({4, "text4", int64_t(4), 5})}),
             exceptions::invalid_request_exception,
-            exception_predicate::message_equals("User Defined Type value contained too many fields (expected 3, got 4)"));
+            exception_predicate::message_contains("contained too many fields (expected 3, got 4)"));
```

Comment:
Why this change?

Reply:
The message changed because I moved validating `user_type` to `abstract_type::validate` (see the next comment).
It now contains `user_type::name()` which is very long and user type specific, so I changed it to `contains()`.

Reply:
ok